conf/routes

```diff
@@ -26,7 +26,10 @@ PUT     /api2/atom/:id/asset-active     controllers.Api2.setActiveAsset(id)
 PUT     /api2/atom/:id/pluto-id         controllers.Api2.setPlutoId(id)
 
 # user uploaded videos
-POST    /api2/atom/:atomId/uploads      controllers.UploadController.create(atomId)
+GET     /api2/uploads                   controllers.UploadController.list(atomId)
+POST    /api2/uploads                   controllers.UploadController.create
+DELETE  /api2/uploads/:id               controllers.UploadController.stop(id)
+POST    /api2/uploads/:id/_credentials  controllers.UploadController.credentials(id, key)
```

Comment:
Why with `_` could we just have `/api2/uploads/:id/credentials/:key` or could the key be sent as a header?